src/app/components/source/Source.js

```diff
@@ -173,6 +173,105 @@ const SourceContainer = Relay.createContainer(SourceComponent, {
               }
             }
           },
+          log(first: 10000) {
```

Comment:
Seems that this is under directly under `source`, not `project_source`. Shouldn't it be under `project_sorce`, just like it is for `project_media`?